CentralComputing/I2CManager.cpp

```diff
@@ -137,16 +140,25 @@ std::shared_ptr<I2CData> I2CManager::refresh() {
   int16_t value = 0;
   
   if (!set_i2c_addr(i2c_fd, addr)) {
-    return false;
+    //TODO: @Anshul, add error here.
```

Comment:
@anshulshah957 Add error generation here. It's not hard, But you should see how to do it.